corpus/definitions.txt

```diff
@@ -333,6 +333,8 @@ object C {
         with C[T]
 }
 
+class D(c: String) extends E(c) with F
```

Comment:
Could you please add the original test case from #56:

```scala
class MyClass extends Potato() with Tomato
```

Which I believe tripped things over by having empty parens in there

Reply:
Done - https://github.com/tree-sitter/tree-sitter-scala/pull/139/commits/ed65e637e1ced2def1de308faed3a2f82dcdf452